### ISSUE TITLE
COCO support with SSD300 BatchNorm

### ISSUE THREAD

**Issue by Deepanshu2017** [amdegroot/ssd.pytorch#74]
Hello,

I am working on a college project which requires SSD implementation with COCO. Since this implementation of SSD is by far the best according to my understanding, are you guys planning to release COCO trained model? I see it on README in "still to come" section by how long will it take?

I would love to help but I do not have much access to GPU so this is why I cannot train the implementation on my own.

Thank you.

**Comment by amdegroot** [comment]
It isn't a difficult thing to do and I know exactly what needs to be done, its just been a matter of finding time.  I will hopefully be able to work on it this weekend, but will keep you posted.  If you do by chance implement it before I do, feel free to post a PR.

**Comment by chauhan-utk** [comment]
@Deepanshu2017 Hey, did you work on the COCO dataset for SSD with this implementation?

**Comment by FostorHUNT** [comment]
@amdegroot hey, I am also waiting for your COCO version SSD. Is there an elementary version code for training SSD on COCO set?

**Comment by tsly123** [comment]
@FostorHUNT , you can consider this temporary solution. I get the weight *.caffemodel from [here](https://github.com/weiliu89/caffe/tree/ssd) and then convert to torch. But there are some errors and I dont know how to do with it yet.
